wpilib/wpilib/resource.py

```diff
@@ -40,15 +40,15 @@ def _reset():
                 if ref is None:
                     continue
                 obj = ref()
-                if obj is not None and hasattr(obj, 'free'):
-                    obj.free()
+                if obj is not None and hasattr(obj, 'close'):
```

Comment:
This needs to allow free and close.

Reply:
if both are defined, can close handle being called twice?

Reply:
It should. But, I think just call one or the other, preferring close.